packages/sanity/src/core/comments/components/pte/comment-input/Editable.tsx

```diff
@@ -22,7 +24,7 @@ const EditableWrapStack = styled(Stack)(() => {
 export const StyledPopover = styled(Popover)(() => {
   return css`
     // Position the Popover relative to the @
-    transform: translateX(6px); // todo: improve
+    transform: translate(6px, 6px); // todo: improve
```

Comment:
Ideally we would configure this through a prop on the `Popover` component ([`offset`](https://floating-ui.com/docs/offset)), but `@sanity/ui` doesn't support this yet.

Reply:
I'll look into this as a separate thing once this is merged